tests/lib/test_Repo_fsck.py

```diff
@@ -151,6 +150,33 @@ def test_fsck_yaml_invalid(caplog, repo):
         assert i in caplog.text
 
 
+contents = ["type: value",
```

Comment:
Please also add a test where the "reserved" pseudo key is in the value, and thus should not fail. For example:

```
key: serial
```
and the more perverse:
```
key: "model: value"
```